docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -41,23 +49,45 @@ Synopsis
         RangePolicy(const RangePolicy&) = default;
         RangePolicy(RangePolicy&&) = default;
 
-        inline RangePolicy();
+        RangePolicy();
+
+        // since 4.3
+        RangePolicy( member_type work_begin
+                   , member_type work_end );
+
+        // since 4.3
+        RangePolicy( member_type work_begin
+                   , member_type work_end
+                   , ChunkSize chunk_size );
+
+        // since 4.3
+        RangePolicy( const execution_space & work_space
+                   , member_type work_begin
+                   , member_type work_end );
+
+        // since 4.3
+        RangePolicy( const execution_space & work_space
+                   , member_type work_begin
+                   , member_type work_end
+                   , ChunkSize chunk_size );
 
+        // until 4.3
         template<class ... Args>
-        inline RangePolicy( const execution_space & work_space
-                          , const member_type work_begin
-                          , const member_type work_end
-                          , Args ... args);
+        RangePolicy( const execution_space & work_space
+                   , member_type work_begin
+                   , member_type work_end
+                   , Args ... args );
 
+        // until 4.3
         template<class ... Args>
-        inline RangePolicy( const member_type work_begin
-                          , const member_type work_end
-                          , Args ... args);
+        RangePolicy( member_type work_begin
+                   , member_type work_end
+                   , Args ... args );
 
```

Comment:
Do we even want to bother mentioning the previous interface?

Reply:
I was a little torn on this.  I'll remove it.  The only reason to keep it is to match compilation errors with the implementation (even though we don't guarantee any particular implementation).